mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1407,7 +1407,11 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		ResourceLabels: expandStringMap(d, "resource_labels"),
 	}
 
-	if v, ok := d.GetOk("enable_shielded_nodes"); ok {
+	// shielded nodes is computed and optional yet serverside
+	// default is true. Forcing true here esentially serves
+	// as a default false but is unavoidable due to how
+	// computed and GetOk work together.
+	if v, _ := d.GetOk("enable_shielded_nodes"); true {
```

Comment:
Can we just remove the check altogether and use `d.Get`?

Reply:
Yep, I suppose I wanted to leave the if statement in there to identify that it was irregular and forcibly divergent behavior, but the comment serves that purpose just fine